src/test/java/nimbus/command/DeleteCommandTest.java

```diff
@@ -0,0 +1,52 @@
+package nimbus.command;
+
+import nimbus.ui.TaskList;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeleteCommandTest {
+    @Test
+    public void testDeleteCommandOneTask() {
```

Comment:
perhaps can use the other convention instead of camel case for test case names? (unitBeingTested_descriptionOfTestInputs_expectedOutcome)